Assets/Scripts/Combat/CreatureStats.cs

```diff
@@ -77,6 +79,11 @@ public int GetMaxActionPoints()
         return 5 + GetEffectiveSpeed() / 10;
     }
 
+    public float GetConcentrationPoints()
```

Comment:
This should be renamed PercentConcentration or possibly GetConcentrationPercent.